Sources/ThirdParty/JSONWebToken+Okta/macOS/RSAPKCS1VerifierMacOS.swift

```diff
@@ -0,0 +1,28 @@
+/*
+* Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+* The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+*
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*
+* See the License for the specific language governing permissions and limitations under the License.
+*/
+
+import Foundation
+
+public struct RSAPKCS1VerifierMacOS: RSAPKCS1VerifierProtocol {
+    public let hashFunction: SignatureAlgorithm.HashFunction
+    public let key: RSAKey
+
+    public init(key: RSAKey, hashFunction: SignatureAlgorithm.HashFunction) {
+        self.hashFunction = hashFunction
+        self.key = key
+    }
+
+    public func verify(_ input: Data, signature: Data) -> Bool {
+        // TODO: implement
+        return true
```

Comment:
IMO - we should return `false` instead of `true` if this functionality isn't ready. That way we do not provide the false illusion that we're successfully able to verify it.

Reply:
Fixed.